dolphinscheduler-common/src/main/java/org/apache/dolphinscheduler/common/utils/OSUtils.java

```diff
@@ -460,23 +460,23 @@ public static int getProcessID() {
     }
 
     /**
-     * check memory and cpu usage
+     * Check memory and cpu usage is overload the given thredshod.
      *
-     * @param maxCpuloadAvg maxCpuloadAvg
+     * @param maxCpuLoadAvg  maxCpuLoadAvg
      * @param reservedMemory reservedMemory
-     * @return check memory and cpu usage
+     * @return True, if the cpu or memory exceed the given thredshod.
      */
-    public static Boolean checkResource(double maxCpuloadAvg, double reservedMemory) {
+    public static Boolean checkOverload(double maxCpuLoadAvg, double reservedMemory) {
```

Comment:
```suggestion
    public static boolean checkOverload(double maxCpuLoadAvg, double reservedMemory) {
```

It'd be more clear if we can rename to something like `isOverloaded`

Reply:
Thanks for your suggestion, agree with you.